### DISCUSSION TITLE
interactsh

### DISCUSSION THREAD

**Discussion by nirsarkar** [projectdiscovery/nuclei#2512] [General]
Version: 2.7.7
[VER] Could not register to http://oast.online: could not make register request: POST https://oast.online/register giving up after 1 attempts: Post "https://oast.online/register": dial tcp 167.99.69.236:443: i/o timeout, retrying with http
[VER] Could not register to oast.online

custom templates load then its problem showing pls fix it .

**Comment by ehsandeep** [comment]
https://github.com/projectdiscovery/nuclei/issues/2513#issuecomment-1230070508
